src/libraries/System.Collections/tests/Generic/List/List.Generic.Tests.Misc.cs

```diff
@@ -203,10 +203,10 @@ public IEnumerable<T> ConstructTestList(T[] items)
 
             #region GetRange
 
-            public void BasicGetRange(T[] items, int index, int count)
+            public void BasicGetRange(T[] items, int index, int count, bool useSlice)
             {
                 List<T> list = new List<T>(items);
-                List<T> range = list.GetRange(index, count);
+                List<T> range = useSlice ? list[index..(index + count)] : list.GetRange(index, count);
```

Comment:
Minor, but I'd prefer it if the tests used the Slice method by name. Otherwise, in the future if the C# lowering changed, we'd lose test coverage for the method.

Reply:
Hmm. I specifically wanted them to use this syntax to verify that the end-to-end experience for slicing is working. I'd prefer to either leave it as is, or add a third version.

Reply:
Please change it.  If you'd like to add additional dedicated tests that use the syntax, that's fine.